Objects/bytearrayobject.c

```diff
@@ -1186,6 +1186,69 @@ bytearray_endswith(PyByteArrayObject *self, PyObject *args)
     return _Py_bytes_endswith(PyByteArray_AS_STRING(self), PyByteArray_GET_SIZE(self), args);
 }
 
+/*[clinic input]
+bytearray.removeprefix as bytearray_removeprefix
+
+    prefix: Py_buffer
+    /
+
+Return a copy of the bytearray with a given prefix removed if present.
+
+If the bytearray starts with the prefix, return b[len(prefix):].
```

Comment:
I like to remind the type of prefix, since currently we don't use type annotations:

```suggestion
If the bytearray starts with the prefix string, return b[len(prefix):].
```

Same remark for other docstrings.